travis/binaries/travis-test.sh

```diff
@@ -21,9 +21,8 @@ ln -s $TRAVIS_BUILD_DIR $UPSTREAM_BUILD_DIR
 
 # Test we can parse Hackage
 # Note: no $TEST_OPTIONS as this isn't tasty suite
-if [ -f ~/.cabal/config ]; then
+cabal update # fetch 01-index.tar
```

Comment:
I see, that was a temporary band-aid. Can we do `user-config init` here instead? That should be faster.

Reply:
We need `update`, `hackage-tests` need index file.

Reply:
... should I add that as a comment?

Reply:
No harm in that.